### PR TITLE
YLP-22 Review content of pdf

### PR DESCRIPTION
**Pull request by fatb38**

### CODE REVIEW

File: packages/viz/src/utils/settings/diabeloopData.js [hunk 1]
```diff
@@ -144,15 +145,10 @@ export function getDeviceInfosData(device) {
...
-  const {
-    level,
-    width,
-  } = opts;
-
+export function getDeviceParametersData(parameters, { level, width }) {
   const heading = {
     text: t("Parameters"),
-    subText: t("level {{level}}", { level }),
+    subText: `- ${level === 1 ? t("Current") : t("Advanced")}`,
```
Comment (franck-fourel) [inline packages/viz/src/utils/settings/diabeloopData.js:151]:
```suggestion
    subText: (level !== 1) ? `- ${t("Advanced")}` : "",
```

File: locales/en/translation.json [hunk 2]
```diff
@@ -105,6 +105,7 @@
   "Correct Above": "Correct Above",
   "Correction": "Correction",
   "Correction factor": "Correction factor",
+  "Current": "Current",
```
Comment (franck-fourel) [inline locales/en/translation.json:108]: I would not use it
```suggestion
```

File: locales/en/translation.json [hunk 3]
```diff
@@ -88,6 +87,7 @@
   "Carb Ratios": "Carb Ratios",
   "Carbohydrates": "Carbohydrates",
   "Carbs": "Carbs",
+  "Cartridge expiration": "Cartridge expiration",
```
Comment (franck-fourel) [inline locales/en/translation.json:90]: why don't you re-use the existing translation ?
"Pump cartridge expiration date"
```suggestion
```

File: packages/viz/src/utils/settings/diabeloopData.js [hunk 4]
```diff
@@ -181,3 +177,98 @@ export function getDeviceParametersData(parameters, opts) {
...
+    label: t("Manufacturer"),
+    value: pump.manufacturer,
+  }, {
+    label: t("Serial Number"),
+    value: pump.serialNumber,
+  }, {
+    label: t("Pump version"),
+    value: pump.swVersion,
+  }, {
+    label: t("Cartridge expiration"),
```
Comment (franck-fourel) [inline packages/viz/src/utils/settings/diabeloopData.js:215]:
```suggestion
    label: t("Pump cartridge expiration date"),
```

**Review by franck-fourel — CHANGES_REQUESTED** [review]
couple of comments
But this first review looks good for me

File: locales/es/translation.json [hunk 5]
```diff
@@ -58,10 +58,9 @@
   "Avg. Daily Total Insulin": "Prom. total de insulina diario",
   "Avg. Glucose ({{bgSourceLabel}})": "Prom. glucemia ({{bgSourceLabel}})",
   "BG": "Glucemia",
-  "BG Distribution": "BG Distribution",
   "BG Log": "Registro de glucemia",
   "BG Target": "Intervalo ideal de glucemia",
-  "BG distribution": "BG distribution",
+  "Time in Range": "Time in Range",
```
[the remark this answers is not in the document]
Comment (fatb38) [reply locales/es/translation.json:63]: didn't see it 😅

File: packages/viz/src/utils/settings/diabeloopData.js (hunk 4 above)
Comment (fatb38) [reply packages/viz/src/utils/settings/diabeloopData.js:215]: didn't see it too !!

**Review by franck-fourel — APPROVED** [review]

**Review by franck-fourel — COMMENTED** [review]
@fatb38 can youplease update the changelog so that we can merge this PR as the cleansing part of ylp-22 ?

**Pull request merged**